File: Corporate-Classifieds/PointsMicroservice/src/main/java/com/cts/pointsmicroservice/model/Offer.java
package com.cts.pointsmicroservice.model;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;



@Getter
@Setter
@AllArgsConstructor
@ToString
public class Offer {
	
	private int id;
	private String name;
	private String description;
	private String category;
	private Date openDate;
	private Date engagedDate;
	private Date closedDate;
	private int likes;
}

File: Corporate-Classifieds/classifieds-portal/src/app/model/Offer.ts
//offer schema for Offer Object
export class Offer{
    constructor(
        public id:number,
        public name:string,
        public description:string,
        public category:string,
        public openDate:Date,
        public engagedDate:Date,
        public closedDate:Date,
        public likes:number
    ){}
}